### PR TITLE
Nakadi api review 20160216

### PR DESCRIPTION
**Pull request by jkliff**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/113" target="_blank">ReviewNinja

File: api/nakadi-event-bus-api.yaml [hunk 1]
```diff
@@ -1029,7 +1024,8 @@ definitions:
           If not explicitly set, default value will respect the definition of the `EventType.category`.
         type: array
         items:
-          $ref: '#/definitions/EventValidationStrategy'
+          type: string
+        default: ['schema-validation']
```
Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: I think we call it `event-body-must-respect-schema`. Should we change it here in the API or in our code?

Comment (jkliff) [inline api/nakadi-event-bus-api.yaml]: Changed in the code now

**Comment by rcillo** [comment]
:shipit:

**Pull request merged**